tracer/test/benchmarks/Benchmarks.Trace/AgentWriterBenchmark.cs

```diff
@@ -32,7 +32,7 @@ static AgentWriterBenchmark()
             var sources = new CompositeConfigurationSource(new[] { overrides, GlobalConfigurationSource.Instance });
             var settings = new TracerSettings(sources);
 
-            var api = new Api(new FakeApiRequestFactory(settings.Exporter.AgentUri), statsd: null, updateSampleRates: null, partialFlushEnabled: false);
+            var api = new Api(new FakeApiRequestFactory(settings.Exporter.AgentUri), statsd: null, updateSampleRates: null, partialFlushEnabled: false, propagateProcessTags: true);
```

Comment:
This will probably skew the benchmarks a little - it will be good to see the results, but we should be aware that we _expect_ them to be impacted. Also this is another case where injecting would be preferable, because then you're not measuring the processtags initialization as part of this benchmark